src/cmd/manage_nsfs.js

```diff
@@ -78,28 +80,17 @@ Bucket Options:
     --config_root <dir>         (default config.NSFS_NC_DEFAULT_CONF_DIR)   Configuration files path for Noobaa standalon NSFS.
 `;
 
-function print_usage() {
-    console.warn(HELP);
-    console.warn(USAGE.trimStart());
-    console.warn(ARGUMENTS.trimStart());
-    console.warn(ACCOUNT_OPTIONS.trimStart());
-    console.warn(BUCKET_OPTIONS.trimStart());
-    process.exit(1);
-}
-
-function print_account_usage() {
-    console.warn(HELP);
-    console.warn(USAGE.trimStart());
-    console.warn(ARGUMENTS.trimStart());
-    console.warn(ACCOUNT_OPTIONS.trimStart());
-    process.exit(1);
-}
-
-function print_bucket_usage() {
+/**
+ * usage will print the usage and help
+ * @param {boolean} print_account will also print the account specific help
+ * @param {boolean} print_bucket will also print the bucket specific help
+ */
+function usage(print_account, print_bucket) {
```

Comment:
```suggestion
function usage({ print_account=false, print_bucket = false }) {
```
I think that the suggested signature will be better understood in the calls to this function

Reply:
I don't like having those defaults.
especially, when it is a boolean and all we do is if(print_account)

Reply:
I think it's better than usage `(false, true)` which is not informative. 
these are just options for usage print, I don't see a reason for having 2 parameters instead of just print_options